hugr-passes/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "hugr-passes"
+version = "0.1.0"
+edition = "2021"
```

```suggestion
version = "0.4.0"
edition = { workspace = true }
rust-version = { workspace = true }

license = { workspace = true }
readme = "README.md"
documentation = "https://docs.rs/hugr-passes/"
homepage = { workspace = true }
repository = { workspace = true }
description = "Compiler passes for Quantinuum's HUGR"
keywords = ["Quantum", "Quantinuum"]
categories = ["compilers"]
```
Let's use the same major/minor version for all packages, to avoid confusion.

---

In that case I can add `version` under `[workspace.package]` in the top-level `Cargo.toml` and use `{workspace = true }` in both packages?

---

Do we want to have to make a new release of `hugr` whenever we do a new release of `hugr-passes`? This shouldn't be necessary.

---

I was thinking about having a matching minor number (0.x), even if the patch version varies.
But it may get desynchronised too, so I guess it's fine to leave it as 0.1.0.

The releases are done synchronously, but only for packages with changes.
See [this PR](https://www.github.com/trillium-rs/trillium/pull/639) for an example on how this looks.